src/io.c

```diff
@@ -96,13 +109,7 @@ unsigned char io_event(unsigned char channel MARK_UNUSED)
 	case SEPROXYHAL_TAG_TICKER_EVENT:
 		UX_TICKER_EVENT(G_io_seproxyhal_spi_buffer, {
 			TRACE("timer");
-			if (timeout_cb)
-			{
-				timeout_callback_fn_t* callback = timeout_cb;
-				// clear first if cb() throws
-				timeout_cb = NULL;
-				callback(UX_ALLOWED);
-			}
+			HANDLE_UX_TICKER_EVENT(UX_ALLOWED);
```

Comment:
I am not sure I like passing UX_ALLOWED (which is a macro) as an argument to a macro. In this case it should be okay but ...

Reply:
actually, UX_ALLOWED is not a macro itself, but a variable injected by the UX_TICKER_EVENT macro